### PR TITLE
Changing Database environment + data generation script

### PR DESCRIPTION
**Pull request by CloverJoy**
- Schema and script for PostgreSQL
- Change the get request query to PSQL database
- Data generation script for bulk data
- Front end is working and connected to s3

### CODE REVIEW

File: database/pgscript.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+FROM '/Users/hezki96/dev/Picture-Service/database/csv/items.csv'
+CSV HEADER;
+
+COPY pictures(picid, fkitemid, largePics, thumbnails)
+FROM '/Users/hezki96/dev/Picture-Service/database/csv/pictures.csv'
+CSV HEADER;
+
+--GET REQUEST QUERY--
+
+SELECT description, largePics, thumbnails FROM items INNER JOIN pictures ON itemid = fkitemid WHERE itemid = 1;
```
Comment (mikegunyan) [inline database/pgscript.sql:36]: I can really see whats going on here...  Good work!

**Review by mikegunyan — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -56,10 +57,15 @@
...
     "msw": "^0.25.0",
+    "node-fetch": "^2.6.1",
     "nodemon": "^2.0.7",
+    "objects-to-csv": "^1.3.6",
+    "pg": "^8.5.1",
+    "pg-hstore": "^2.3.3",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "sequelize": "^6.5.0",
```
Comment (juanoxmar) [inline package.json]: uninstall sequelize & sequlize-cli

**Review by juanoxmar — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -56,7 +56,11 @@
     "mongodb": "^3.6.3",
     "mongoose": "^5.11.11",
     "msw": "^0.25.0",
+    "node-fetch": "^2.6.1",
```
Comment (juanoxmar) [inline package.json:59]: node-fetch & axios accomplish the same thing. Please select one and uninstall the other

**Review by juanoxmar — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (juanoxmar) [reply package.json:59]: ok checked with you. node-fetch is ok

**Pull request merged**